app/controllers/images_controller.rb

```diff
@@ -0,0 +1,74 @@
+class ImagesController < ApplicationController
+  before_action :set_image, only: [:show, :edit, :update, :destroy]
+
+  # GET /images
+  # GET /images.json
+  def index
+    @images = Image.all
+  end
+
+  # GET /images/1
+  # GET /images/1.json
+  def show
+  end
+
+  # GET /images/new
+  def new
+    @image = Image.new
+  end
+
+  # GET /images/1/edit
+  def edit
+  end
+
+  # POST /images
+  # POST /images.json
+  def create
+    @image = Image.new(image_params)
+
+    respond_to do |format|
+      if @image.save
+        format.html { redirect_to @image, notice: 'Image was successfully created.' }
+        format.json { render :show, status: :created, location: @image }
+      else
+        format.html { render :new }
+        format.json { render json: @image.errors, status: :unprocessable_entity }
+      end
+    end
+  end
+
+  # PATCH/PUT /images/1
+  # PATCH/PUT /images/1.json
+  def update
+    respond_to do |format|
+      if @image.update(image_params)
+        format.html { redirect_to @image, notice: 'Image was successfully updated.' }
```

Comment:
Metrics/LineLength: Line is too long. [85/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)